packages/vanilla/src/basics/form-elements/range/range.js

```diff
@@ -51,12 +51,18 @@ class Range extends Core {
 
   setMax(maxValue) {
     this._findDOMEl('.hig__range__field', this.el).setAttribute('max', maxValue);
-    this._findDOMEl('.hig__range__field__range-values', this.el).dataset.rangeMax = maxValue;
+    const dataset = this._findDOMEl('.hig__range__field__range-values', this.el).dataset;
+    if (dataset) {
```

Comment:
Included this since the enzyme-mounted component does not implement [dataset](https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/dataset).